_maps/chilledstation.json

```diff
@@ -0,0 +1,76 @@
+{
+	"version": 1,
+	"map_name": "ChilledStation",
+	"map_path": "map_files/ChilledStation",
+	"map_file": "ChilledStation.dmm",
+	"space_ruin_levels": 0,
+	"space_empty_levels": 0,
+	"planetary": 1,
+	"shuttles": {
+		"cargo": "cargo_chilled",
+		"ferry": "ferry_fancy",
+		"whiteship": "whiteship_box",
+		"emergency": "emergency_box"
+	},
+	"traits": [
+		{
+			"Up": 1,
+			"Mining": true,
+			"Linkage": null,
+			"Gravity": true,
+			"Ice Ruins Underground": true,
+			"Baseturf": "/turf/open/lava/plasma/ice_moon",
+			"No Parallax": true
+		},
+		{
+			"Down": -1,
+			"Up": 1,
+			"Mining": true,
+			"Linkage": null,
+			"Gravity": true,
+			"Ice Ruins Underground": true,
+			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
+			"No Parallax": true
+		},
+		{
+			"Down": -1,
+			"Up": 1,
+			"Mining": true,
+			"Linkage": null,
+			"Gravity": true,
+			"Ice Ruins Underground": true,
+			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
+			"No Parallax": true
+		},
+		{
+			"Down": -1,
+			"Up": 1,
+			"Mining": true,
+			"Linkage": null,
+			"Gravity": true,
+			"Ice Ruins Underground": true,
+			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
+			"No Parallax": true
+		},
+		{
+			"Down": -1,
+			"Mining": true,
+			"Linkage": null,
+			"Gravity": true,
+			"Ice Ruins": true,
+			"Weather_Snowstorm": true,
+			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
+			"No Parallax": true
+		}
+	],
+	"minetype": "none",
```

Comment:
```suggestion
		{
			"Down": -1,
			"Up": 1,
			"Mining": true,
			"Linkage": null,
			"Gravity": true,
			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
			"No Parallax": true
		},
		{
			"Down": -1,
			"Up": 1,
			"Mining": true,
			"Linkage": null,
			"Gravity": true,
			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
			"No Parallax": true
		},
		{
			"Down": -1,
			"Mining": true,
			"Linkage": null,
			"Gravity": true,
			"Weather_Snowstorm": true,
			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
			"No Parallax": true
		}
	],
	"minetype": "none",
```
![image](https://user-images.githubusercontent.com/34697715/215232210-8699b6c3-2f65-495c-84c7-52728d6460ea.png)

The ruin loader is wasting time on every initialize because the z-level is flagged to have these ruins, but you (deliberately) haven't placed any valid areas for it to place stuff into